moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -175,10 +175,10 @@ const ProjectView = () => {
   let [searchParams, setSearchParams] = useSearchParams();
   const location = useLocation();
   const activeTab = useActiveTabIndex(searchParams.get("tab"));
-  const previousFilters = location.state?.filters;
-  const allProjectsLink = !!previousFilters
-    ? `/moped/projects?filter=${previousFilters}`
-    : "/moped/projects";
+  const { queryString: previousProjectListViewQueryString } = location.state;
+  const allProjectsLink = !previousProjectListViewQueryString
+    ? "/moped/projects"
+    : `/moped/projects${previousProjectListViewQueryString}`;
```

Comment:
Bigger update here, instead of extending this to pass the state of `isOr` along with the state of `filters`, I updated this to get all query strings using React Router so that this will support any future params too.

Reply:
niiiice!